plugins/modules/inventory_intent.py

```diff
@@ -2720,39 +2796,40 @@ def get_diff_deleted(self, config):
                         execution_details = self.get_task_details(task_id)
 
                         if 'success' in execution_details.get("progress"):
-                            self.msg = "Device Deleted Successfully from Cisco DNA Center"
                             self.status = "success"
+                            self.msg = "Device deleted successfully from Cisco Catalyst Center"
+                            self.log(self.msg, "INFO")
                             self.result['changed'] = True
                             self.result['response'] = execution_details
                             break
                         elif execution_details.get("isError"):
                             self.status = "failed"
                             failure_reason = execution_details.get("failureReason")
                             if failure_reason:
-                                self.msg = "Device Deletion get failed because of {0}".format(failure_reason)
+                                self.msg = "Device deletion get failed because of {0}".format(failure_reason)
                             else:
-                                self.msg = "Device Deletion get failed."
-                            self.log(self.msg)
+                                self.msg = "Device deletion get failed."
```

Comment:
Specify the device, please check the above comment

Reply:
added the device ip.